config/triplea/schemas/lobby_server.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Lobby server set",
+  "description": "A set of lobby servers",
+  "type": "array",
+  "items": {
+    "title": "Lobby server",
+    "description": "A lobby server",
+    "type": "object",
+    "properties": {
+      "version": {
+        "description": "The lobby server engine version",
+        "type": "string",
+        "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$"
+      },
+      "host": {
+        "description": "The lobby server host",
+        "type": "string",
+        "anyOf": [
+          { "format": "host-name" },
+          { "format": "ipv4" },
+          { "format": "ipv6" }
+        ]
+      },
+      "port": {
+        "description": "The TCP port on which the lobby server accepts connections",
+        "type": "number",
+        "minimum": 0,
+        "maximum": 65535
+      },
+      "message": {
+        "description": "The message displayed when a user connects to an available lobby server",
+        "type": "string"
+      },
+      "error_message": {
```

Comment:
I allowed this property to be of type `null` because it looks like it's simply blanked-out when the server is available as opposed to being removed.  One other possibility is to make it optional but always of type `string` when it is defined.  This will probably require some testing to ensure the code behaves the same when `error_message` is removed (a casual inspection of `LobbyServerProperties#isServerAvailable()` seems to suggest the code will handle this).